src/App.jsx

```diff
@@ -1,14 +1,23 @@
+
 import {
   createBrowserRouter,
   RouterProvider,
 } from "react-router-dom";
-import Home from "./pages/home"
```

Comment:
Since we don't use the home component anymore, should we consider to delete the file as well? (not just the import line)